public_html/wp-content/plugins/pattern-directory/includes/patterns.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace WordPressdotorg\Pattern_Directory\Patterns;
+use const WordPressdotorg\Pattern_Directory\Pattern_Post_Type\POST_TYPE;
+
+/**
+ * The main class for handling information about patterns.
+ *
+ * @package WordPressdotorg\Pattern-directory
+ */
+class Patterns {
```

Comment:
Do we want to have a helper-type class to contain some of the pattern-related queries/helper functions like this? 
I stole this approach from the plugin directory where there is a `PluginDirectory` class.

Please advise.